test/e2e/feature/feature.go

```diff
@@ -557,9 +557,6 @@ var (
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
 	Vsphere = framework.WithFeature(framework.ValidFeatures.Add("vsphere"))
 
-	// TODO: document the feature (owning SIG, when to use this feature for a test)
-	WatchList = framework.WithFeature(framework.ValidFeatures.Add("WatchList"))
```

Comment:
dropping this in favor of the framework.WithFeatureGate incantation which is automatically informed by feature gate stability level and default state